frontend/src/app/event-list/event-list.component.ts

```diff
@@ -7,9 +11,20 @@ import { Component, OnInit } from '@angular/core';
 })
 export class EventListComponent implements OnInit {
 
-  constructor() { }
+  public geoEvents$: Observable<IGeoEvent[]> | null = null;
+
+  constructor(private geoEventsService: GeoEventsService) { }
 
   ngOnInit(): void {
+    this.geoEvents$ = this.geoEventsService.getAll();
+  }
```

Comment:
There's no need to set the observable during `ngOnInit` since you're not setting it back to null anywhere.

What you do handle in there are the subscriptions created from that observable.

```suggestion
  readonly geoEvents$ = this.geoEventsService.getAll();

  constructor(private readonly geoEventsService: GeoEventsService) { }
```

No need to add typing to the variable at this place imo. It keeps it more readable and if we make small functions for everything, it'll be typed checked at the first next step.